app/webpacker/components/TeamsCommitteesCouncils/GroupPage.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { Header } from 'semantic-ui-react';
+import { Button, Header } from 'semantic-ui-react';
 import EmailButton from '../EmailButton';
-import { apiV0Urls } from '../../lib/requests/routes.js.erb';
+import { apiV0Urls, contactRecipientUrl } from '../../lib/requests/routes.js.erb';
 import I18n from '../../lib/i18n';
 import useLoadedData from '../../lib/hooks/useLoadedData';
 import Loading from '../Requests/Loading';
 import Errored from '../Requests/Errored';
 import RolesTable from './RolesTable';
 
+function contactRecipientUrlForGroup(group) {
+  if (group.metadata.friendly_id === 'wrt') {
+    return contactRecipientUrl('results_team');
+  }
+  return contactRecipientUrl('communications_team');
```

Comment:
Lots of hard-coded strings. At the very least, the references to `results_team` and `communications_team` should be exposed through some `ContactForm` constants.

Ideally, this would use the new "preferred contact mode" feature like so: (pseudo-code)
```
if (group.metadata.preferred_contact_mode === 'private') { // perhaps store string as constant somewhere
  return contactRecipientUrl(group.metadata.friendly_id);
}
return contactRecipientUrl(someFallback);
```

The parameter to pass to `contactRecipientUrl` is a bit tricky here, because we generally use friendly IDs for T/Cs but the contact form expects humanized values instead (might be worth changing this?!)
Also not sure what `fallback` should be or whether we even need it. If a T/C set their preferred communication mode to `private`, they should definitely have a corresponding contact form recipient. So there really shouldn't be the case where we even need a fallback.

Reply:
Now I'm thinking should I change the contact recipient to friendly ID, because it's safer to do this now than later. What do you think?